MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -3016,8 +3023,10 @@ public MediaStream GetMediaStream(IEnumerable<MediaStream> allStream, int? desir
         }
 
         public static string GetHwScaleFilter(
+            string hwScalePrefix,
```

Comment:
Is there any or will there be any scale filter does not use the name `scale`?

Reply:
It is unlikely to happen. When new hw scale filters are added, they need to follow the existing naming logic: `{scale,vpp}_{libname}`.